src/components/theme-engine/index.tsx

```diff
@@ -22,9 +25,18 @@ export class Container extends React.Component<Properties> {
   }
 
   static mapActions(_props: Properties): Partial<Properties> {
-    return {};
+    return { setViewMode };
   }
 
+  componentDidMount() {
+    const viewMode = this.getSavedViewMode(savedViewMode) === 'true' ? ViewModes.Light : ViewModes.Dark;
+    this.props.setViewMode(viewMode);
+  }
+
+  getSavedViewMode = (savedViewMode: string): string => {
```

Comment:
this method returns either true or false as string, which is not what the name of the function describe.

can wee change it to `isLightModeIsSaved` (this is just an example). also to get rid of the logic of `'true'` in `componentDidMount`